include/libcyphal/transport/transport.hpp

```diff
@@ -62,6 +87,8 @@ class ITransport : public IRunnable
     ///
     virtual cetl::optional<ArgumentError> setLocalNodeId(const NodeId node_id) noexcept = 0;
 
+    virtual void setTransientErrorHandler(TransientErrorHandler handler) = 0;
```

Comment:
```suggestion
    virtual void setTransientErrorHandler(const TransientErrorHandler& handler) = 0;
```

The docs are missing, should we expect them later?

Reply:
>Proper docs will come in the next PR when there will be more experience. For now this is just blueprint.